.github/ISSUE_TEMPLATE/BUG_REPORT.yml

```diff
@@ -20,7 +20,7 @@ body:
           required: true
         - label: Is this only a single bug? Do not put multiple bugs in one issue.
           required: true
-        - label: Is this a UI / front end issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo.
+        - label: Is this a backend issue? Use the [lemmy-ui](https://github.com/LemmyNet/lemmy-ui) repo for UI / frontend issues.
           required: false
```

Comment:
You can change it to required true then.

Reply:
Yes right!